server/src/routes/index.ts

```diff
@@ -101,6 +118,91 @@ router.get('/api-docs', swaggerUI.setup(openapiSpecification));
  *             description: Previous Courses
  *             items:
  *               $ref: '#/definitions/Course'
+ *   Credentials:
```

Comment:
If Swagger allows for it, these definitions should be made in the file that uses them, so in this case `routes/session.ts`.